use-timescale/continuous-aggregates/OLD_real-time-aggregates.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Real-time aggregates
+excerpt: Real-time aggregates combine pre-aggregated data with the most recent raw data for up-to-date results
+products: [cloud, mst, self_hosted]
+keywords: [continuous aggregates, real-time aggregates]
+---
+
+import CaggsRealTimeHistoricalDataRefreshes from 'versionContent/_partials/_caggs-real-time-historical-data-refreshes.mdx';
+
+# Real time aggregates
+
+<Highlight type="important">
+In Timescale&nbsp;1.7 and later, real time aggregates are enabled by default.
+You do not need to manually enable real-time aggregation.
+</Highlight>
+
+Real-time aggregates use a live query to combine aggregated data and
```

Comment:
While talking to Gina the other day we found that "on-the-fly" may be a better term than "live" to describe the behavior. While technically the same, it feels like on-the-fly may capture the "it's going to be run every single time" better - may be a non-native feeling though ;-)